src/apps/alpha/alpha.c

```diff
@@ -0,0 +1,128 @@
+#include "buttons.h"
+#include "common.h"
+
+Button sw2,sw3;
+
+uint8_t counter=0;
+uint8_t location=0;
+uint8_t word[2][8]={0};
+
+int get_char(char c){
+    if(c>='A' && c<='Z'){
+        return c-'A';
+    }
+    else if(c>='a' && c<='z'){
+        return c-'A'-6;
+    }
+    else{
+        return -1; //caracter no soportado
```

Comment:
Fijate de implementar la logica para agregar los numeros y los simbolos, algo asi
```suggestion
    else{
        switch(c){
            case ':': return 10;
            case '!': return 11;
            case '?': return 12;
            case '¿: return 13;
            case '¡': return 14;
            case ' ': return 15;
            default: return -1; //caracter no soportado
        }
```
Y mira el otro comentario respecto a los caracteres ¿ y ¡